runtime-image/Dockerfile.in

```diff
@@ -28,21 +28,21 @@ RUN wget https://storage.googleapis.com/python-interpreters/latest/interpreter-3
 
 # Add Google-built interpreters to the path
 ENV PATH /opt/python3.7/bin:/opt/python3.6/bin:/opt/python3.5/bin:/opt/python3.4/bin:$PATH
-RUN update-alternatives --install /usr/local/bin/python3 python3 /opt/python3.6/bin/python3.6 50
-RUN update-alternatives --install /usr/local/bin/pip3 pip3 /opt/python3.6/bin/pip3.6 50
+RUN update-alternatives --install /usr/local/bin/python3 python3 /opt/python3.7/bin/python3.7 50
+RUN update-alternatives --install /usr/local/bin/pip3 pip3 /opt/python3.7/bin/pip3.7 50
 
 # Upgrade pip (debian package version tends to run a few version behind) and
 # install virtualenv system-wide.
-RUN /usr/bin/pip install --upgrade -r /resources/requirements.txt && \
-    /opt/python3.4/bin/pip3.4 install --upgrade -r /resources/requirements.txt && \
-    rm -f /opt/python3.4/bin/pip /opt/python3.4/bin/pip3 && \
-    /opt/python3.5/bin/pip3.5 install --upgrade -r /resources/requirements.txt && \
-    rm -f /opt/python3.5/bin/pip /opt/python3.5/bin/pip3 && \
-    /opt/python3.6/bin/pip3.6 install --upgrade -r /resources/requirements.txt && \
-    rm -f /opt/python3.6/bin/pip /opt/python3.6/bin/pip3 && \
-    /opt/python3.7/bin/pip3.7 install --upgrade -r /resources/requirements.txt && \
-    rm -f /opt/python3.7/bin/pip /opt/python3.7/bin/pip3 && \    
-    /usr/bin/pip install --upgrade -r /resources/requirements-virtualenv.txt
+RUN /usr/bin/pip install --upgrade -r /resources/requirements.txt
+RUN /opt/python3.4/bin/pip3.4 install --upgrade -r /resources/requirements.txt
+RUN rm -f /opt/python3.4/bin/pip /opt/python3.4/bin/pip3
+RUN /opt/python3.5/bin/pip3.5 install --upgrade -r /resources/requirements.txt
+RUN rm -f /opt/python3.5/bin/pip /opt/python3.5/bin/pip3
+#RUN /opt/python3.6/bin/pip3.6 install --upgrade -r /resources/requirements.txt
+#RUN rm -f /opt/python3.6/bin/pip /opt/python3.6/bin/pip3
+RUN /opt/python3.7/bin/pip3.7 install --upgrade -r /resources/requirements.txt
+RUN rm -f /opt/python3.7/bin/pip /opt/python3.7/bin/pip3
+RUN /usr/bin/pip install --upgrade -r /resources/requirements-virtualenv.txt
```

Comment:
what's the advantage of separating each command out to its own RUN command?

Reply:
good question.  When all the install commands are on one line, an error in any of them fails the whole command and it is nearly impossible to debug.  I separated them in order to debug the issues, and then I thought it might be good to keep them separated.

OH!, but I see a problem.  I still have python 3.6 commented out!